src/components/style.css

```diff
@@ -59,6 +59,9 @@ button#rotate {
   .hide-password {
     display: none;
   }
+  .hide-encrypt {
```

Comment:
Now that there's a second thing we want to hide from the card on print, it might make sense to make it generic instead of adding a second input-specific selector that does the same thing.

We could rename `.hide-password` to `.no-print` and use that single class on both the hide-password and encryption-mode inputs. Further, now that each of those sections is wrapped in a `<div>` you could simply apply that class to the surrounding `<div>` instead of every label/input element. e.g.:

```html
<div className="no-print">
  <label for="hide-password-checkbox" ... </label>
  <input
    type="checkbox"
    id="hide-password-checkbox"
    ...
  />
</div>

<div className="no-print">
  <label ...>Encryption: </label>
  <input
    type="radio"
    id="encrypt-none"
    ...
  />
  ....
</div>
```